ci/Dockerfile

```diff
@@ -0,0 +1,45 @@
+ARG TENDERMINT_VERSION=0.34.9
+ARG GAIA_VERSION=4.2.1
```

Comment:
It may be better to use Gaiad v5 or v6 (mainnet hub-4 is at [v6](https://github.com/cosmos/gaia/releases/tag/v6.0.0)).

Reply:
I can currently only get this working for Gaiad v5.0.5 (that's the latest image we have from [`cephalopodequipment`](https://hub.docker.com/r/cephalopodequipment/gaiad/tags) in the v5.x series). v6.0.0 fails at the point where I try to add the generated user key to the `basecoin-0` chain via Hermes:

```
Configuring Hermes...
2021-12-23T14:10:10.979891Z TRACE ThreadId(01) deserialized the encoded pub key into a ProtoAny of type '/cosmos.crypto.secp256k1.PubKey'
Success: Added key 'testkey' (cosmos1va7pcr6gt6p8vvyexrmzwrm6k8uuc7rq2z96w0) on chain ibc-0
Generating user key...
{"name":"user","type":"local","address":"cosmos177vu4gpafpzd9f39kwc64m3tevk628dr3xe6d0","pubkey":"{\"@type\":\"/cosmos.crypto.secp256k1.PubKey\",\"key\":\"A1QTDg0L9pZeHs0WHf73tTjyUBa8UI2Z931CHjUCSCVN\"}","mnemonic":"safe defense winter glide ladder belt clip inform they forest utility record feature cannon deliver stomach will mean destroy border matter sleep gospel involve"}
Adding user key to basecoin-0 chain...
Error: error encoding key: EOF while parsing a value at line 1 column 0
```

I'm assuming the serialization format changed between Gaiad v5 and v6?

Reply:
Just tested this and it seems gaiad-v0.6.0 writes the output of the `keys add` command to `stderr` instead of `stdout`, so the `user_seed.json` file is empty leading to this problem.